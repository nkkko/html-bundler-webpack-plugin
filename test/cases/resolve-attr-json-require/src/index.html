<!DOCTYPE html>
<html>
<head>
  <title>Test</title>
  <script src="./main.js" defer="defer"></script>
</head>
<body>
  <h1>Hello World!</h1>

  <a id="test-elm" href="#" data-image='{ "alt":"picture", "imgSrc": require("./kiwi.png"), "bgImage": { "imgSrc": require("./lemon.png") } }'>
    <img src="./image.png" >
  </a>
</body>
</html>