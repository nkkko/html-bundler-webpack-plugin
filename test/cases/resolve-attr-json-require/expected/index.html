<!DOCTYPE html>
<html>
<head>
  <title>Test</title>
  <script src="js/main.bundle.js" defer="defer"></script>
</head>
<body>
  <h1>Hello World!</h1>

  <a id="test-elm" href="#" data-image='{ "alt":"picture", "imgSrc": "img/kiwi.da3e3cc9.png", "bgImage": { "imgSrc": "img/lemon.7b66be8e.png" } }'>
    <img src="img/image.697ef306.png" >
  </a>
</body>
</html>